packages/connectivity/src/http-agent/http-agent.ts

```diff
@@ -107,10 +134,14 @@ function getCertificateOption(destination: Destination): Record<string, any> {
     );
   }
 
-  const options = trustAllOptions(destination)({});
+  let options = trustAllOptions(destination)({});
+  options = trustStoreOptions(options, destination);
```

Comment:
General: these functions `trustAllOptions`, `trustStoreOptions` and `certificateOptions` are not named in accordance with our style guide. This could be improved, #boyscoutprinciple.